app/src/main/java/com/cris/nvh/framgiaproject/MiniMediaPlayer.java

```diff
@@ -73,16 +65,43 @@ public ImageView getTrackImage() {
 		return mTrackImage;
 	}
 
+	public PlayMusicService getService() {
+		return sService;
+	}
+
+	public void setService(PlayMusicService service) {
+		sService = service;
+	}
+
 	public void pause() {
-		mButtonPlay.setImageResource(R.drawable.ic_play);
+		sService.requestPause();
+		mImagePlay.setBackgroundResource(R.drawable.ic_play);
 	}
 
 	public void play() {
+		sService.requestStart();
+		mImagePlay.setBackgroundResource(R.drawable.ic_pause);
 	}
 
 	public void next() {
+		sService.requestChangeTrack(0);
 	}
 
 	public void previous() {
+		sService.requestChangeTrack(0);
```

Comment:
```suggestion
		sService.previousTrack(0);
```
Sao next vs previous lại giống nhau về phương thức. em nên tách thành 2 phương thức thực thi 2 công việc khác nhau nhé.